modules/opensearch/opensearch.install

```diff
@@ -20,6 +20,8 @@ function opensearch_migrate_settings() {
     'ting_recommendation_url' => 'opensearch_recommendation_url',
     'ting_enable_logging' => 'opensearch_enable_logging',
     'ting_cache_lifetime' => 'opensearch_cache_lifetime',
+    'ting_search_autocomplete_suggestion_url' => 'opensearch_search_autocomplete_suggestion_url',
+    'ting_search_autocomplete_method' => 'opensearch_search_autocomplete_method',
```

Comment:
:+1: but you might have forgotten some variables. What about `opensearch_search_autocomplete_settings`?

Reply:
I see that some settings have been moved - some have not. That is probably why it is not appropriate to delete `ting_search_autocomplete_settings` entirely.

We still need to migrate (parts of) it somehow.